test/contextmenu.spec.js

```diff
@@ -171,6 +175,10 @@ describe('ContextMenu component', () => {
     it('root menu element without veil viewport limit.', () => {
       // When it is rendering outside of viewport,
       // size of menu is set to 180 as a default
+      HTMLDivElement.prototype.getBoundingClientRect = jest.fn(() => {
```

Comment:
리턴문을 빼고 바로 객체를 리턴해도 되겠습니다.
```suggestion
      HTMLDivElement.prototype.getBoundingClientRect = jest.fn(() => ({right: 20, bottom: 20}));
```

Reply:
저도 바로 객체를 반환하는 방식으로 작성했었는데, 린트 룰의 `'arrow-body-style': ['error', 'as-needed', { requireReturnForObjectLiteral: true }]`에 의해 위 형태로 변경되었습니다.

해당 린트 룰 삭제 후 변경사항 반영하면 될까요?